src/lib/autocomplete/autocomplete.ts

```diff
@@ -126,6 +126,16 @@ export class MatAutocomplete extends _MatAutocompleteMixinBase implements AfterC
   }
   private _autoActiveFirstOption: boolean;
 
+  /**
+   * Whether the width of the autocomplete panel is set to 'auto'.  When 'false', the width of the
+   * panel will match the width of the host.
+   */
+  @Input()
+  get panelWidthAuto(): boolean { return this._panelWidthAuto; }
```

Comment:
Rather than directly controlling if the panel width is auto, would it make more sense to take in a string which can be auto or any CSS sizing value?  This would cover a few more use cases.

We could even have a default of something like 'match' to match the host's width.

```ts
@Input() panelWidth: string = 'match';
```

Reply:
IMO we should avoid having magic strings like these in APIs. We can make it so by default it matches the input width, unless the consumer passed in a truthy width.

Reply:
use a string literal if you don't want a regular string

Reply:
@crisbeto good point.  Using `match` is a magic string.  Really, I just wanted to make it a bit more flexible than a `boolean` would be.